### PR TITLE
WIP - chore(global): updating styles to give some space

### PR DESCRIPTION
**Pull request by benouat**
This is really an early WIP, I open the PR so we can discuss ideas.

The main idea is to update the overall aspect to give more space a bit everywhere.
Titles need some rooms, sections, paragraphs too.

Sidebar sections needs some space too.
I'll continue to dig this a bit, and comment with screen capture as soon as I have something to share.

### CODE REVIEW

**Comment by icfantv** [comment]
@benouat, I appreciate the effort here.  let's talk first before you get too in the weeds and go crazy with code changes.  I'll be back at work Monday but won't be able to start looking at L&F until Monday night US time.

I'm ok with some of the ideas here, but not others.

**Comment by benouat** [comment]
@icfantv Thank's for your comment !
Just for you to know, I am not going into the wild without an guidance here... I am sitting in the same room as @pkozlowski-opensource and have been discussing these changes a lot with him :)

**Comment by icfantv** [comment]
@benouat.  apologies on my part.  he explained and i shouldn't have responded without being able to provide more detail.

File: demo/src/app/components/shared/api-docs/api-docs.component.html [hunk 1]
```diff
@@ -1,12 +1,15 @@
 <div class="api-doc-component">
   <h2>
+    {{apiDocs.className}}
     <a href="https://github.com/ng-bootstrap/ng-bootstrap/tree/master/{{apiDocs.fileName}}"
        target="_blank"
        angulartics2On="click"
        angularticsEvent="Demo HTML Content"
-       angularticsCategory="{{ demoTitle }}">{{apiDocs.className}}</a>
+       angularticsCategory="{{ demoTitle }}">
```
Comment (icfantv) [inline demo/src/app/components/shared/api-docs/api-docs.component.html]: I don't know that I love the text link here.  Maybe an icon?  A GH icon?  I'm not married to the component title being a hyperlink, but do agree that we should link to the source on GH....just not sure how.

**Comment by icfantv** [comment]
@benouat we need some sort of physical divider between component headers, e.g., in the accordion demo, something above `NgbPanel` to delineate it from `Inputs`, `Outputs`, and/or `Methods`.  This may be due to it using the same font/heading.

Leaving the component headings as H2 and moving the section headings to H4 looks good, IMO - this may obviate the need for a physical divider.

Thoughts?

**Comment by wesleycho** [comment]
This needs to be rebased - I think the stripping out of the angulartics plugin caused this.

**Comment by pkozlowski-opensource** [comment]
@benouat this got a bit stalled and would require some more work to cover items listed in #825. As such I'm going to close this one for now. Let's re-open when you've got time to work on this.

**Pull request closed without merging**